code/game/turfs/simulated/floor/fancy_floor.dm

```diff
@@ -113,6 +114,32 @@
 		icon_state = "basalt[rand(0, 12)]"
 		set_basalt_light(src)
 
+
+/turf/open/floor/grass/fakebasalt //Heart is not a real planeteer power
+	name = "aesthetic volcanic flooring"
+	desc = "Safely recreated turf for your hellplanet-scaping"
+	icon = 'icons/turf/floors.dmi'
+	icon_state = "basalt"
+	floor_tile = /obj/item/stack/tile/basalt
+	ore_type = /obj/item/weapon/ore/glass/basalt
+	slowdown = 0
+
+/turf/open/floor/grass/fakebasalt/Initialize()
+	..()
+	if(prob(15))
+		icon_state = "basalt[rand(0, 12)]"
+		set_basalt_light(src)
+
+/turf/open/floor/grass/fakebasalt/attackby(obj/item/C, mob/user, params)
```

Comment:
But you ARE uprooting the grass in the tile (its a organic thing) but the ash tile doesn't have any organic content, if anybody wants to PR over that later be my guest.

Yeah, i know its the same proc, i copy pasted it for **_peace of mind_**, i literally do not mind nor care and i think just changing the description is enough since it works & compiles. Its really not worth worrying about since i don't think anybody will inherit from this and instead deviate to call upon the grass directly, and if people want to add more ores (the sand content dropped) they can.

> fake/real snow & ice tiling are things for later.